content/advent-2015/generate-free-tls-certs-with-lego.md

```diff
@@ -121,27 +119,27 @@ Now with all the paperwork out of the way, we can finally get to the exciting st
 
 ## Obtaining Certificates
 
-You can obtain one certificate per domain name (`client.ObtainCertificates`), or you can obtain a single SAN certificate (`client.ObtainSANCertificate`) which is valid for multiple domain names. SAN certificates are most commonly used for the "www." variant of the domain name. Typically, you'll want different certificates for different sites.
+You can obtain a certificate with a call to `client.ObtainCertificate`. It can issue both regular and SAN certificates, depending on how many names you pass in. SAN certificates are most commonly used for the "www." variant of the domain name. Typically, you'll want different certificates for different sites. You'll get back a `CertificateResource` which contains a certificate, private key, and some metadata.
 
 ```go
-certificates, failures := client.ObtainCertificates([]string{"mydomain.com", "www.mydomain.com"}, true)
+certResource, failures := client.ObtainCertificate([]string{"example.com", "www.example.com"}, true, nil)
 if len(failures) > 0 {
-    // At least one cert request failed; but some may have succeeded.
-    // Make sure to save the certs and keys for those that succeeded.
+    // At least one domain failed to verify, but others may have succeeded.
+    // If there were any failures, no certificate will be returned.
     for domain, err := range failures {
         log.Printf("[%s] %v", domain, err)
     }
 }
 ```
 
-The second argument is whether lego should bundle the intermediate certificates for us. Usually this is `true` unless you have a good reason.
+The first argument is the list of domains to include in the certificate; a challenge will be solved for each domain name. The second argument is whether lego should bundle the intermediate certificates for us; this is always `true` unless you have a good reason. The last argument is a private key to use; this is usually `nil` since lego will generate one for you, but you should provide one if you implement HPKP.
```

Comment:
https://en.wikipedia.org/wiki/HTTP_Public_Key_Pinning

But frankly, only readers who already know what it should be messing with it ;) It's dangerous...